index.js

```diff
@@ -14,6 +14,7 @@ const NahmiiContract = require('./lib/contract');
 const InsufficientFundsError = require('./lib/insufficient-funds-error');
 const NahmiiEventProvider = require('./lib/event-provider');
 const ClusterInformation = require('./lib/cluster-information');
+const {FeesClaimant} = require('./lib/claim');
```

Comment:
Er dette riktig? Hadde forventet
`const FeesClaimant = require('./lib/fees-claimant');`

Reply:
Your snippet would work too. I foresee we'll possibly add a claimants in the future, e.g. for NII from contract `RevenueTokenManager` or from instant settlements. Hence the more general require of `./lib/claim`.